examples/jupyter/pincell_depletion.ipynb

```diff
@@ -0,0 +1,1028 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "pyplot.plot(time, xe135, label=\"Xe135\")\n",
+    "pyplot.xlabel(\"Time [d]\")\n",
+    "pyplot.ylabel(\"Number of atoms - Xe135\");"
+   ]
+  },
+  {
+
```

Comment:
Suggested change: "Use time steps that are too short and you may..."
Same thing for long case